product_attachment_zipped_download/views/product_product_views.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8" ?>
```

Comment:
Technically, this (and the following file) isn't a view, so it may go to `data` into a file `ir_actions_server_data.xml` file, but not strictly needed.